.github/workflows/ci_tests_backward.yaml

```
@@ -0,0 +1,105 @@
+# This workflow installs PyGMT and runs tests with old GMT versions
+
+name: GMT Backward Tests
```

Not so sure if 'backward' is a good word, how about 'legacy' (https://en.wikipedia.org/wiki/Legacy_system)? Maybe we need to ask a native English speaker :slightly_smiling_face:

```suggestion
name: GMT Legacy Tests
```

---

*Legacy* sounds good and let's wait for some inputs for native English speakers.

---

Done in be81f365240de56636f4d3d324a3417f00090e9c